ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/lacp.j2

```diff
@@ -0,0 +1,15 @@
+{# eos - lacp #}
+{% if lacp is arista.avd.defined %}
+!
+{%     if lacp.port_id.range.begin is arista.avd.defined and lacp.port_id.range.end is arista.avd.defined %}
+lacp port-id range {{ lacp.port_id.range.begin }} {{ lacp.port_id.range.end }}
+{%     endif %}
+{%     if lacp.rate_limit.default is arista.avd.defined(false) %}
+no lacp rate-limit default
+{%     else %}
```

Comment:
please change this into `{% elif lacp.rate_limit.default is arista.avd.defined(true) %}`
If someone just wants to set the lacp system-priority, we should not generate extra config.

Reply:
https://eos.arista.com/eos-4-18-0f/incoming-lacpdu-rate-limit/, lacp rate-limit is set by default. I will only keep the no lacp rate-limit command and remove the else condition.